core/nlu/model/SnipsNlu.py

```diff
@@ -105,8 +107,10 @@ def train(self):
 			'language': self.LanguageManager.activeLanguage,
 		}
 
+		#TODO: I suppose this could be
+		# for file in self._cachePath.glob(f'*_{self.LanguageManager.activeLanguage}.json')
 		for file in self._cachePath.glob('*.json'):
-			if not f'_{self.LanguageManager.activeLanguage}' in file.stem:
+			if f'_{self.LanguageManager.activeLanguage}' not in file.stem:
 				continue
```

Comment:
the language would always be in the end so this could be added to the glob right?

Reply:
Correct also, didn't think about it